go/vt/binlog/binlogplayer/dbclient.go

```diff
@@ -37,18 +38,20 @@ func (dc *DBClient) handleError(err error) {
 	}
 }
 
+// Connect connects to a db server
 func (dc *DBClient) Connect() error {
 	params, err := dbconfigs.MysqlParams(dc.dbConfig)
 	if err != nil {
 		return err
 	}
-	dc.dbConn, err = mysql.Connect(params)
+	dc.dbConn, err = sqldb.Get()(params)
```

Comment:
Instead of sqldb.Get()(params), I think it would be more idiomatic to provide a function sqldb.NewConn(params) or sqldb.Connect(params).